ior.spec

```diff
@@ -96,17 +94,23 @@ $MPI_MAN/man1/*
 EOF
 %endif
 
-
 %if (0%{?suse_version} >= 1)
 %files
 %{_bindir}/*
 %{_defaultdocdir}/%{name}/
 %{_mandir}/man1/*
 %else
+%find_lang mdtest --with-man
 %files -f files.mpich
+%files -f mdtest.lang
 %endif
 
 %changelog
+* Fri Jan 12 2024 Dalton A. Bohning <dalton.bohning@intel.com> - 4.0.0-1
```

Comment:
Let's do some datestamp updating here at some point since this is now 10 days ago.